### PR TITLE
network.py module: add get interface speed function

### PR DESCRIPTION
**Pull request by mkalfon**

### CODE REVIEW

File: rrmngmnt/network.py [hunk 1]
```diff
@@ -683,3 +683,22 @@ def is_connective(self, ping_timeout=20.0):
...
+        """
+        Get network interface speed
+
+        Args:
+            interface (str): Interface name
+
+        Returns:
+            str: Interface speed or empty string in case of failure
+        """
+        rc, out, _ = self._cmd(
```
Comment (myakove) [inline rrmngmnt/network.py]: show the error in the logger.error.

File: rrmngmnt/network.py [hunk 2]
```diff
@@ -683,3 +683,17 @@ def is_connective(self, ping_timeout=20.0):
...
+        Get network interface speed
+
+        Args:
+            interface (str): Interface name
+
+        Returns:
+            str: Interface speed
+        """
+        return self._cmd(
+            ["cat", "/sys/class/net/{iface}/speed".format(iface=interface)]
```
Comment (nellyc) [inline rrmngmnt/network.py]: this is problematic on vms (including nested hosts)
getting: "cat: /sys/class/net/eth0/speed: Invalid argument"

worth to check why or at least document it

**Comment by mkalfon** [comment]
@tareqalayan @nellyc could you please merge the change? I need it to skip tier 3 runs to avoid failures.

**Comment by mkalfon** [comment]
@lukas-bednar can we publish it to pypi?

**Comment by lukas-bednar** [comment]
@mkalfon your change will be part of [0.1.14](https://github.com/rhevm-qe-automation/python-rrmngmnt/milestone/13) release, use that link for status. Right now I am waiting for one more PR to be included.

**Pull request merged**